picasso/src/main/java/com/squareup/picasso3/ImageDecoder.java

```diff
@@ -0,0 +1,34 @@
+package com.squareup.picasso3;
+
+import android.graphics.Bitmap;
+import android.graphics.drawable.Drawable;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import java.io.IOException;
+import okio.BufferedSource;
+
+public interface ImageDecoder {
```

Comment:
This should be renamed so we don't conflict with `android.graphics.ImageDecoder`.

Reply:
ImageFormatDecoder?